includes/html/common/alert-log.inc.php

```diff
@@ -37,6 +37,10 @@
     'Warning' => 5,
 ];
 
+if (Auth::user()->hasGlobalAdmin()) {
+    $admin_verbose_details = '<th data-column-id="verbose_details" data-sortable="false">Details</th>';
```

Comment:
This only prevents the link from showing up, the details are still in the html below.  You'll need to add a check down there too.